classes/Collection.js

```diff
@@ -162,6 +145,19 @@ class Collection {
         }
       })
       await sendTree(newGitTree, currentCommitSha, this.siteName, this.accessToken, commitMessage);
+
+      // Update collection.yml in newCollection with newCollection name
+      const collectionConfig = new CollectionConfig(this.accessToken, this.siteName, newCollectionName)
+      const { content: configContent, sha: configSha } = await collectionConfig.read()
+      const configContentObject = yaml.safeLoad(base64.decode(configContent))
+      const newConfigContentObject = {
+        collections: {
+          [newCollectionName]: configContentObject.collections[oldCollectionName]
+        }
+      }
+      const newConfigContent = base64.encode(yaml.safeDump(newConfigContentObject))
+      await newConfig.update(newConfigContent, configSha)
```

Comment:
this should be `collectionConfig.update` instead of `newConfig.update`

Reply:
good catch, fixed with 59a99fd